### ISSUE TITLE
Black screen video - Ubuntu 17.10

### ISSUE THREAD

**Issue by felipemartins210** [baedert/corebird#787]
![screenshot from 2017-11-14 01-45-23](https://user-images.githubusercontent.com/8534899/32761946-8266cfc6-c8dd-11e7-8d79-e540ebb94f33.png) 

when i try to watch something.

**Comment by baedert** [comment]
And what version is that? And how did you install it? And what does `corebird --gst-debug-level=3` print when  you open a video? What ffmpeg version do you have installed (in light of #776)?

**Comment by felipemartins210** [comment]
![captura de tela de 2017-11-17 15-19-28](https://user-images.githubusercontent.com/8534899/32960058-c4e0b144-cbaa-11e7-9607-6ba2eaa0c739.png)

Ubuntu 17.10, I have installed by using the Ubuntu system store.

**Comment by Vistaus** [comment]
By screenshotting the output, it's basically unusable. Copy the full output from the terminal and paste it here.

**Comment by j605** [comment]
As the error says, install `gst-plugins-bad`.

**Comment by mr-blobbyyy** [comment]
Testing out various distros again I ran into this same problem. **gstreamer1.0-plugins-bad** was already installed but I traced the issue to not having **gstreamer1.0-libav** installed -- which was odd. My best guess is that this is the result of **not** checking the "install non-free software" box during Ubuntu's installation procedure. ([Or it may be that 17.10 is still using v1.6.](https://packages.ubuntu.com/artful/corebird))

@baedert, probably worth bookmarking this for the Ubuntu people.
